### ISSUE TITLE
[bug] species not available from pubMLST.org

### ISSUE THREAD

**Issue by mtruglio** [bactopia/bactopia#273]
**Description**
Hi, for some reason when trying to add Cutibacterium acnes to the datasets, by running:

bactopia datasets --ariba "vfdb_core,card" --species "Cutibacterium acnes" --include_genus --limit 100 --cpus 40

Everything goes well, except I get "Cutibacterium acnes is not available from pubMLST.org, skipping". Which is actually not true, see https://pubmlst.org/organisms/cutibacterium-acnes

Is there a way around this? Alternatively, how can I set up the schema manually, after downloading the data from pubmlst?

Thanks

Bactopia: v.2.0.1
OS: Ubuntu 20.04
Environment: conda

***EDIT***
Apparently, it's due to the fact that the species changed name from Propionibacterium Acnes to Cutibacterium Acnes, but the pubmlst api was not updated accordingly.

**Comment by rpetit3** [comment]
Looks like I need to update my list of available schemas. I'll get this updated in the next version

**Comment by rpetit3** [comment]
Hi @mtruglio 

I looked into this. Seems like the issue is _Cutibacterium acnes_ is not available from http://pubmlst.org/data/dbases.xml

I'm going to look into the to see if there are alternate ways to programmatically retrieve the current schemas

Cheers

**Comment by rpetit3** [comment]
Looked more into this, looks like its still _Propionibacterium acnes_ in the dbases.xml. I've reached out to the pubMLST curator. Will hopefully have an update soon

**Comment by rpetit3** [comment]
Haha, just saw your edit! apologies

**Comment by rpetit3** [comment]
This has been updated on the pubmlst side, and will be fixed in the next version (fix here https://github.com/bactopia/bactopia/commit/e0566286bb860e66f1b0da7f3245be2dd9c9005e)

**Comment by rpetit3** [comment]
If you like to give these changes a try, I have a latest build on my personal anaconda channel.

```
mamba create -n test-bactopia -c rpetit3 -c conda-forge -c bioconda bactopia
```

**Comment by mtruglio** [comment]
Thanks Robert, I'll test it asap and let you know!

Cheers


________________________________

**Comment by rpetit3** [comment]
I've released v2.0.2 (https://github.com/bactopia/bactopia/releases/tag/v2.0.2) which includes a fix for this issue, but also many other changes! I'm going to go ahead and close this for now, please feel free to reopen!

If you have a moment, could you take a few minutes to fill out a [Bactopia Usage Survey](https://forms.gle/PTgUxEBwCsCdbQjr7)? It really helps me direct the direction of Bactopia's development! Thanks!
